pkg/yttlibrary/url.go

```diff
@@ -245,8 +250,10 @@ func (uv *URLValue) User() starlark.Value {
 	return uu
 }
 
+func (uu *URLUser) Type() string { return "@ytt:url.user" }
```

Comment:
since we are getting into "first classing" type, should we also be adding Len(), Truthy()?